spec/lib/payment_calculator/ecf/contract/service_fee_calculations_spec.rb

```diff
@@ -8,12 +8,14 @@ class DummyClass
 
 describe ::PaymentCalculator::Ecf::Contract::ServiceFeeCalculations do
   it "performs calculations of the service fees" do
-    band_a = double("Band Double", per_participant: 996.00)
+    band_a = double("Band Double", per_participant: 996.00,
+                                   number_of_participants_in_this_band: 2000,
+                                   deduction_for_setup?: true)
     contract = double("Contract Double", recruitment_target: 2000, set_up_fee: 149_651.00, band_a: band_a)
     call_off_contract = DummyClass.new({ contract: contract })
 
-    expect(call_off_contract.service_fee_total.round(2)).to eq(647_149.00)
-    expect(call_off_contract.service_fee_monthly.round(0)).to eq(22_315.00)
-    expect(call_off_contract.service_fee_per_participant.round(0)).to eq(324.00)
+    expect(call_off_contract.service_fee_total(band_a).round(0)).to eq(647_149.00)
```

Comment:
Why do you round the output here?

Reply:
We round as per requirement in https://docs.google.com/spreadsheets/d/1wGrFuwoyZJcQlyjnFxQD-5H8cWljSfbc/edit#gid=1504585790 

```Where a price is required, the figure will be rounded to the nearest pound```

Reply:
@plisovin Should that be in the calculator itself?

Reply:
It's outside of the scope of this ticket, there is another one https://dfedigital.atlassian.net/browse/CPDTP-106